### PR TITLE
Misc. updates

### PR DESCRIPTION
**Pull request by ten1seven**
Small requested updates that address the following tickets: #311, #306, #308, #307, #309, #312, and #310

### CODE REVIEW

File: web/themes/npm/archive-product.php [hunk 1]
```diff
@@ -10,19 +10,19 @@
 $args = array(
     'taxonomy' => $taxonomy,
     'meta_key' => 'appears_on_products',
-    'meta_value' => 1,
+    'meta_value' => 1
```
Comment (ten1seven) [inline web/themes/npm/archive-product.php]: I was getting a weird parse error until I removed the trailing commas in these arrays...

```
Parse error: syntax error, unexpected ')' in /Users/jeremy/Sites/npm/web/themes/npm/archive-product.php on line 42
```

Comment (justinvandenberg) [reply web/themes/npm/archive-product.php]: Hmm, weird. Do you know what version of PHP you're running?

Comment (ten1seven) [reply web/themes/npm/archive-product.php]: `7.2.21`

Comment (mzlock) [reply web/themes/npm/archive-product.php]: Weird. PHP is usually cool with trailing commas. That's actually the preferred standard in PHP to leave trailing commas.

Seems like tailing commas _in function calls_ is new to 7.3 though: https://laravel-news.com/php-trailing-commas-functions

**Pull request merged**